### PR TITLE
Improve docs for Terraform CLI example

### PR DESCRIPTION
**Pull request by gibbleyg**

### CODE REVIEW

File: examples/terraform/complex_cli_only/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+> [!IMPORTANT]
+> If your module can be used directly as a root module, consider following the [simple_cli_only](../simple_cli_only) example instead.
+
+This example meets the
+[Marketplace requirements](https://cloud.google.com/marketplace/docs/partners/vm/configure-terraform-deployment#requirements_for_custom_terraform_modules)
+for a Terraform **CLI Only** module
+
+If the `examples/marketplace_test` folder exists, marketplace will validate your
+module by executng:
```
Comment (linhlam-kc) [inline examples/terraform/complex_cli_only/README.md:9]: Typo. It should be "executing"

File: examples/terraform/complex_cli_only/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+```
+terraform -chdir=examples/marketplace_test init
+terraform -chdir=examples/marketplace_test plan -var project_id=<test-project>
+```
+
+The module in `examples/marketplace_test` must reference your module in the root
+directory. This test module can be used to create resources that
```
Comment (linhlam-kc) [inline examples/terraform/complex_cli_only/README.md:17]: Just a recommendation. We should add a code snippet for the Partner with limited knowledge of Terraform to understand how to reference the module in the root directory.

```
# The test module references the module in the root directory
module "test" {
  source = "../.."
}
```

**Review by linhlam-kc — APPROVED** [review]
LGTM when the typo is fixed. I also added a minor recommendation in the README file.

File: examples/terraform/complex_cli_only/README.md (hunk 2 above)
Comment (gibbleyg) [reply examples/terraform/complex_cli_only/README.md]: Good suggestion! Added

**Pull request merged**